HowToIntegrateWithSpringBoot.md

```diff
@@ -1,4 +1,4 @@
-# Integrate your Spring-boot application with WSO2 Identity Server
+logoutUrl("/applogout)# Integrate your Spring-boot application with WSO2 Identity Server
```

Comment:
This change seems to be mistake. Can you please correct it?

Reply:
fixed